operations/mimir-mixin/alerts/alerts.libsonnet

```diff
@@ -161,8 +161,8 @@ local utils = import 'mixin-utils/utils.libsonnet';
           alert: $.alertName('MemcachedRequestErrors'),
           expr: |||
             (
-              sum by(%s, name, operation) (rate(thanos_memcached_operation_failures_total[1m])) /
-              sum by(%s, name, operation) (rate(thanos_memcached_operations_total[1m]))
+              sum by(%s, name, operation) (rate(thanos_cache_operation_failures_total[1m])) /
```

Comment:
I think we should add backward compatibility here. An easy way should be doing something like this (please manually test it to make sure I'm not missing anything):

```
sum by(%s, name, operation) (
  # Backward compatibility.
  rate(thanos_memcached_operation_failures_total[1m])
  or
  rate(thanos_cache_operation_failures_total[1m])
)
/
sum by(%s, name, operation) (
  # Backward compatibility
  rate(thanos_memcached_operations_total[1m])
  or
  rate(thanos_cache_operations_total[1m])
)
```

Reply:
Good point. I will update everywhere I've changed to do the query in a backwards compatible way.